### DISCUSSION TITLE
Cannot apply for GCE (Github Campus Expert) Despite being approved for Github Education Pack

### DISCUSSION THREAD

**Discussion by AnenIsaac** [community/community#137356] [GitHub Education]
### Hi everyone,

I recently completed the verification process for GitHub Student Developer Pack yesterday, after encountering some delays due to document and email issues. However, I have not yet received my benefits, which I understand include eligibility to apply for the GitHub Campus Experts program.

Unfortunately, the deadline to apply for the GitHub Campus Experts program is just a few hours away. Without the benefits, I am unable to proceed with my application. I am deeply passionate about the program and would greatly appreciate any assistance you can provide to expedite the activation of my benefits so that I can submit my application before the deadline.

**Comment by davevad93** [comment]
Hi @AnenIsaac ,

I don't know if your benefits will be available before closing because it usually takes a few days and sometimes in periods with high volume of requests it can take even longer like a couple of weeks. However, applications for the program will open again in February, so you'll have another opportunity to apply in a few months.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @AnenIsaac was approved and your benefits are now available.`

**Reply by AnenIsaac** [reply]
Thanks for the answer @davevad93 , kind of a bummer but I guess that just the way life goes sometimes. I thought there would be something I could do to help speed up the process. Thanks for your quick response. Have a lovely day!

**Reply by davevad93** [reply]
You're welcome @AnenIsaac , don't worry in February you'll have another chance. Have a nice day you too!
